pdns/recursordist/docs/settings.rst

```diff
@@ -1069,7 +1069,7 @@ IP address for the webserver to listen on.
 ``webserver-allow-from``
 ------------------------
 -  IP addresses, comma separated
--  Default: 0.0.0.0, ::/0
+-  Default: 0.0.0.0, ::/0 before 4.1.0, 127.0.0.1,::1 since 4.1.0
```

Comment:
in sphinx, the following syntax can be used:

```rst
.. versionchanged:: 4.1.0

  Changed from '0.0.0.0, ::/0'
```

This makes all these changes render consistently and makes it easy to grep for these things in upcoming releases to e.g. remove them :)